src/Blockcore.Indexer/Storage/Mongo/MongoStorageOperations.cs

```diff
@@ -69,11 +69,10 @@ public void ValidateBlock(SyncBlockTransactionsOperation item)
 
                   CreateBlock(item.BlockInfo);
 
-                  ////if (string.IsNullOrEmpty(lastBlock.NextBlockHash))
-                  ////{
-                  ////    lastBlock.NextBlockHash = item.BlockInfo.Hash;
-                  ////    this.SyncOperations.UpdateBlockHash(lastBlock);
```

Comment:
Could you please explain why UpdateBlockHash call was removed? Was it not needed? Happening later?

Reply:
I added an fix to update the Confirmations and PreviousBlockHash values, to the x42 repo and cleaned those changes up (didnt want to commit multiple different things in one PR). I probably thought this function had to do with it, will revert those changes ;) and open another PR.